xml/System.IO/DirectoryInfo.xml

```diff
@@ -3177,9 +3184,19 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>Returns the original path that was passed by the user.</summary>
+        <summary>Returns the original path that was passed by the user. Use the <see cref="P:System.IO.DirectoryInfo.FullName" /> or <see cref="P:System.IO.DirectoryInfo.Name" /> properties for the full path or file/directory name.</summary>
```

Comment:
```suggestion
        <summary>Returns the original path that was passed to the <see cref="T:System.IO.DirectoryInfo" /> constructor. Use the <see cref="P:System.IO.DirectoryInfo.FullName" /> or <see cref="P:System.IO.DirectoryInfo.Name" /> properties for the full path or file/directory name instead of this method.</summary>
```

Reply:
Thanks for the change. This addresses Jeremy's suggestion.